### DISCUSSION TITLE
Separate Users and Things from Clients

### DISCUSSION THREAD

**Discussion by dborovcanin** [absmach/magistrala#2290] [General]
While both Users and Things act as clients, they do not have many properties in common.
For example, user has email, first name, last name, username, password, and potentially metadata such as address and other relevant info.
Thing, on the other hand, has an id, name - relevant mostly for UI - MAC address, private key, serial number, Thing key, and metadata that has nothing to do with users' metadata.

Users and things are naturally separated, and they used to be different entities in the Magistrala. We joined them as Clients so we could group them simply, but that is not relevant at this stage of the project.

**Comment by arvindh123** [comment]
+1
